### DISCUSSION TITLE
Combinando fontes

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#973] [Dicas]
recentemente comecei a fazer uma página simples, mas queria um design bem minimalista e bonito, queria fontes que combinassem entre si e não fossem uma confusão generalizada, então achei [este site](https://fontpair.co/) que te mostra fontes do Google Fonts que combinam entre si.

 Mas se você quer ficar bom nisso, tem um ótimo curso do[ CodeSchool sobre tipografia](http://design.codeschool.com/levels/1), que te ensina tudo que é necessário, e é isso. é apenas uma dica que resolvi deixar aqui. colaborem com as suas se desejarem.

**Comment by kvnol** [comment]
Tipografia ♥

**Comment by NandoSangenetto** [comment]
Adorei o site. Passei para o pessoal de UI/UX, eles ficaram encantados.

**Comment by pedrocss** [comment]
Esse curso da CodeSchool, citado pelo @leandroruel, na verdade se chama [Fundamentals of Design](https://www.codeschool.com/courses/fundamentals-of-design), dá pra assistir a primeira parte sobre Tipografia, é gratuíto. Inclusive tem um link relacionado interessante:

- [Thinking with Type](http://thinkingwithtype.com/) (Tem uma seção ["Combinando fontes"](http://thinkingwithtype.com/letter/#mixing-typefaces), porém é curta).
